caffe2/python/cnn.py

```diff
@@ -521,6 +521,22 @@ def Iter(self, blob_out, **kwargs):
             **kwargs)
         return self.net.Iter(blob_out, blob_out, **kwargs)
 
+    def Accuracy(self, blob_in, blob_out, **kwargs):
```

Comment:
can we move this to model_helper.py instead, as this is not CNN specific function?